docs/testing/platform-limits.md

```diff
@@ -0,0 +1,44 @@
+---
+title: 'Platform limits with Office Scripts'
+description: 'Resource limits and browser support for Office Scripts when used with Excel on the web'
+ms.date: 07/23/2020
+localization_priority: Normal
+---
+
+# Platform limits with Office Scripts
```

Comment:
(Not critical) Should we call it limits and requirements? The cookie thing is more of a requirement...

Reply:
I'm good with that. I'll add that to the title, but leave the TOC as the shortened version